### PR TITLE
fix documentation

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

**Comment by cqc-melf** [comment]
Small documentation fix

**Review by qartik — APPROVED** [review]

File: pytket/qir/conversion/api.py [hunk 1]
```diff
@@ -55,9 +55,9 @@ def pytket_to_qir(
     :type name: str
     :param qir_format: format of the generated qir, default value is binary
     :type qir_format: QIRFormat
-    :param pyqir_0_6_compatibility: converts the output to be compatible with
-        pyqir 0.6, default value false
-    :type pyqir_0_6_compatibility: bool
+    :param wfh: wasm file handler used when creating the circuit.
+      Only needed when there are wasm calls in the circuit.
+    :type wfh: pytket.wasm.WasmFileHandler
```
Comment (cqc-alec) [inline pytket/qir/conversion/api.py]:
```suggestion
    :type wfh: Optional[pytket.wasm.WasmFileHandler]
```

**Review by cqc-alec — DISMISSED** [review]

**Comment by qartik** [comment]
Ah, good catch. I wonder if the documentation generator can just pick the types from the annotations instead of us having to write them and potentially making copy/paste errors.

**Comment by cqc-alec** [comment]
> Ah, good catch. I wonder if the documentation generator can just pick the types from the annotations instead of us having to write them and potentially making copy/paste errors.

I think it's not necessary to include these `:type:` annotations anyway, as the types are annotated and visible in the documentation.

**Comment by cqc-melf** [comment]
I have now removed all the types

**Review by qartik — APPROVED** [review]
Thanks for the quick update!

**Pull request merged**